### PR TITLE
Make less an npm task

### PR DESCRIPTION
**Pull request by Pomax**
removes less processing as gulp task(s) and instead makes it run as an npm script.

this PR builds on top of https://github.com/mozilla/learning.mozilla.org/pull/2126, so the first commit is cruft, actual diff is https://github.com/mozilla/learning.mozilla.org/pull/2127/commits/38a8b674afe66c92bd3269a3bdac585ba2f6c171

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +16,16 @@
     "node": "^4.4.1"
   },
   "scripts": {
-    "app": "npm run l10n && gulp app",
-    "build": "npm run lint && gulp",
+    "app": "npm run prep && npm run l10n && gulp app",
+    "build": "npm run prep && gulp",
     "cleanup": "rm -rf dist && npm run l10n",
     "l10n": "pontoon-to-json",
+
```
Comment (gvn) [inline package.json]: FWI: Blank lines will get nuked when you run `npm i some-package --save`.

**Comment by gvn** [comment]
The task doesn't seem to be re-running on changes to LESS files. Repro by changing something on `index.less` and see that nothing happens.

**Comment by Pomax** [comment]
yeah, we don't have a "watch" task right now, I'm reintroducing that after the nogulp work.

**Comment by gvn** [comment]
Ok, I recommend the bizarrely named https://github.com/paulmillr/chokidar for watching.

**Comment by Pomax** [comment]
I love chokidar. I remember attending a Cascadia.js session on it by the people that came up with it.

**Comment by alanmoo** [comment]
This makes sense to me, no need for watching if this is an intermediate task and Less processing will be handled by webpack.

**Comment by alanmoo** [comment]
P.S. I can't read "chokidar" without thinking about @thealphanerd saying it during his talk at Cascadia.js. Such a fun name.

**Comment by Pomax** [comment]
closing as reviewed and superceded.

**Pull request closed without merging**